docs/source/applications/face_detection.rst

```diff
@@ -15,20 +15,27 @@ Learn more: `https://paperswithcode.com/task/face-detection <https://paperswithc
 
 Using our API you easily detect faces in images as shown below:
 
-.. literalinclude:: ../../../examples/face_detection/main.py
-   :language: python
-   :lines: 36-54
+.. code-block:: python
 
-Check the full example `here <https://github.com/kornia/kornia/tree/master/examples/face_detection/main.py>`_
-or run a real-time application using the camera with this `example <https://github.com/kornia/kornia/tree/master/examples/face_detection/main_video.py>`_.
+    # select the device
+    device = torch.device('cpu')
+    if args.cuda and torch.cuda.is_available():
+        device = torch.device('cuda:0')
 
-The Kornia AI Game
-------------------
+    # load the image and scale
+    img_raw = cv2.imread(args.image_file, cv2.IMREAD_COLOR)
```

Comment:
Use Kornia io

Reply:
These are from changes on the master (merged here), since these configs didn't edit anything on this file, this should be fixed in another PR